.github/workflows/edr-npm-release.yml

```diff
@@ -29,35 +29,37 @@ jobs:
           - host: macos-latest
             target: x86_64-apple-darwin
             build: |
-              yarn build
+              pnpm build
               strip -x *.node
           - host: windows-latest
-            build: yarn build
+            build: pnpm build
             target: x86_64-pc-windows-msvc
           - host: windows-latest
             build: |
-              yarn build --target i686-pc-windows-msvc
-              yarn test
+              pnpm build --target i686-pc-windows-msvc
+              pnpm test
             target: i686-pc-windows-msvc
           - host: ubuntu-latest
             target: x86_64-unknown-linux-gnu
             docker: ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian
             build: |-
               set -e &&
-              yarn build --target x86_64-unknown-linux-gnu &&
+              (wget -qO- 'https://unpkg.com/@pnpm/self-installer' | node) &&
```

Comment:
Did the command fail without installing pnpm? I'm asking, because the source indicates that the [alpine](https://github.com/napi-rs/napi-rs/blob/9d62db47e89f78e94a713ead6978637f1158e3d8/alpine.Dockerfile#L19) and [debian](https://github.com/napi-rs/napi-rs/blob/9d62db47e89f78e94a713ead6978637f1158e3d8/debian.Dockerfile#L37) images have pnpm installed. Maybe it's just a path issue?

Reply:
You are right, fixed!

Reply:
Cheers!